backend/src/main/java/storage/Persistance.java

```diff
@@ -0,0 +1,15 @@
+package storage;
+
+import model.Customer;
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'model.DigitalPart'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)